### ISSUE TITLE
Some question 

### ISSUE THREAD

**Issue by Biho** [inikdom/rnn-speech#24]
Thank you for you sharing code.

I have some question this code,
First, tensorflow CTC loss contain softmax operation. Nevertheless your model contain layer that hidden to numlabel. I think this layer is not need... Is any other reason for this layer?
second is not question. In my working env. , I don't finish my python cosole. so every run time, error is occured. because previous run's graph is maintain. I suggest that main code contain "tf.reset_default_graph() " this code.
